exporter/sentryexporter/README.md

```diff
@@ -7,13 +7,15 @@ For more details about distributed tracing in Sentry, please view [our documenta
 The following configuration options are supported:
 
 - `dsn`: The DSN tells the exporter where to send the events. You can find a Sentry project DSN in the “Client Keys” section of the “Project Settings” section of a Sentry project.
+- `insecure`: If it is set to true, then ssl certificates will not be checked. Useful for test purposes, as well as for Sentry installations deployed in private clouds.
```

Comment:
Can we rename this to something more specific than insecure? Seems like the AWS exporter uses `no_verify_ssl` https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/aws/awsutil/awsconfig.go#L29 which I think better highlights what this config option does.

I would like to also see more emphasis on warning that this is a dangerous option, akin to the docstring for `InsecureSkipVerify` in https://golang.org/pkg/crypto/tls/#Config.

Reply:
Done.

I just searched this repository for the word "insecure" and see that in most cases either "insecure" or "insecure_skip_verify"is used. In rare cases, use "use_insecure", which I think is not convenient at all.